src/React/Redux.purs

```diff
@@ -30,25 +30,22 @@ module React.Redux
   , reducerOptic
   , spec
   , spec'
+  , composeMiddleware
   , applyMiddleware
   , fromEnhancerForeign
   ) where
 
-import Prelude (Unit, (>>=), (<<<), const, pure, id, unit)
-
+import React as React
 import Control.Monad.Eff (Eff)
 import Control.Monad.Eff.Class (class MonadEff, liftEff)
-
 import Data.Either (Either, either)
 import Data.Function.Uncurried (Fn2, Fn3, runFn2, runFn3)
 import Data.Lens (Getter', Lens', Prism', matching, set, to, view)
 import Data.Tuple (Tuple(..), fst)
-
+import Prelude (Unit, const, id, pure, unit, (<<<), (>>=))
 import Unsafe.Coerce (unsafeCoerce)
 
-import React as React
```

Comment:
I think I'd like the `React` import towards the bottom here like it was. Usually, I try to keep all the non-core imports below the core ones. I am considering core imports to be `Prelude`, `Control`, `Data`, etc.